bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkImpl.java

```diff
@@ -58,8 +58,19 @@ public LinkImpl(String url, String target) {
      * @param reference Referenced WCM/DAM entity
      */
     LinkImpl(String url, String target, T reference) {
+        this(url, target, reference, null, null);
+    }
+
+    /**
+     * @param url Link URL
+     * @param target Target
+     * @param reference Referenced WCM/DAM entity
+     * @param linkAccessibilityLabel Accessibility Label
+     * @param linkTitleAttribute Title attribute
+     */
+    LinkImpl(String url, String target, T reference, String linkAccessibilityLabel, String linkTitleAttribute) {
```

Comment:
I'd put the `T reference` argument at the beginning, as all other arguments are part of the link properties